### ISSUE TITLE
Recycler View Items not shown after back

### ISSUE THREAD

**Issue by eschgi** [Arasthel/SpannedGridLayoutManager#20]
Hello,

first of all, thanks for this great library =). I think i've found a bug. Every time when i open a new fragment and then go back to that fragment where i use the SpannedGridLayoutManager no items are shown until i scroll a little bit.

Here my code how i replace my fragments:

```java
private void replaceFragment(@NonNull Fragment fragment) {
	try {
		FragmentManager fragmentManager = getSupportFragmentManager();
		fragmentManager.beginTransaction().replace(R.id.fragment_container, fragment, TAG_CURRENT_FRAGMENT).commit();
		mCurrentFragment = fragment;
	} catch (Exception e) {
		Log.e(TAG, e.getMessage());
	}
}
```

Am I doing something wrong?

Kind regards
Stefan

**Comment by jaumard** [comment]
I think I have same kind of problem, but for me it's only on new fragment, first time the recycler view is rendered it's blank. And when I scroll a little the all list appear, no problem on back navigation on my side.

**Comment by Arasthel** [comment]
Should be fixed on 2.0.3. Sorry for not addressing this earlier.
